og.post_update.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @file
+ * Post-update functions for the Organic groups module.
+ */
+
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\og\OgMembershipInterface;
+
+/**
+ * Apply schema updates on the 'state' field of 'og_membership' entity.
+ */
+function og_post_update_og_membership_state_field(&$sandbox) {
+  // The 'state' base field of the 'og_membership' entity was changed from a
+  // 'string' field to a 'list_string' type field. This implementation of
+  // hook_post_update_NAME() follows the guidance in docs, linked below,
+  // to update the field's storage and resolve the entity field definition
+  // mismatch which would otherwise be reported in the drupal Status Report
+  // page.
+  // @see https://www.drupal.org/docs/drupal-apis/update-api/updating-entities-and-fields-in-drupal-8#s-updating-a-base-field-type
+  /** @var \Drupal\Core\Entity\EntityDefinitionUpdateManagerInterface $definition_manager */
+  $definition_manager = \Drupal::service('entity.definition_update_manager');
+  if (!$definition_manager->needsUpdates()) {
+    // No updates necessary.
+    return t('No entity updates to run.');
```

Comment:
We always use TranslatableMarkup instead of t():
```suggestion
    return new TranslatableMarkup('No entity updates to run.');
```